ocrs/src/preprocess.rs

```diff
@@ -39,10 +39,8 @@ fn greyscale_image<F: Fn(f32) -> f32>(
 
     for y in 0..height {
         for x in 0..width {
-            let mut pixel = 0.;
-            for c in 0..used_chans {
-                pixel += img[[c, y, x]] * chan_weights[c];
-            }
+            let pixel: f32 =
+                (0..used_chans).fold(0.2, |pixel, c| pixel + img[[c, y, x]] * chan_weights[c]);
```

Comment:
The initialization value changed from `0` to `0.2` here. I'd prefer to stick with the for loop as I find it easier to read and more importantly, performance in hot loops has been more predictable in my experience.

Reply:
reverted in a774da5cbd1f0298fafb21cae718058791b2c1ac